src/WebServer_I2C_Scanner.ino

```diff
@@ -176,6 +239,77 @@ String getKnownI2Cdevice(byte address) {
   return result;
 }
 
+int scanI2CbusForDevices( // Utility function for scanning the I2C bus for valid devices, with HTML table output
+        int8_t muxAddr
+      , int8_t channel
+      , int nDevices
+#ifdef FEATURE_I2CMULTIPLEXER
+      , String &excludeDevices
+#endif
+) {
+  byte error, address;
+#ifdef FEATURE_I2CMULTIPLEXER
+  bool skipCheck = false;
+#endif
+
+  for (address = 1; address <= 127; address++)
+  {
+#ifdef FEATURE_I2CMULTIPLEXER
+    if (channel != -1 && excludeDevices.length() != 0) {
+      String toFind = F("|");
+      toFind += String(address);
+      toFind += F("|");
+      skipCheck = (excludeDevices.indexOf(toFind) != -1);
+    }
+    if (!skipCheck) { // Ignore I2C multiplexer and addresses to exclude when scanning its channels
+#endif
+      Wire.beginTransmission(address);
+      error = Wire.endTransmission();
+
```

Comment:
Need a `delay(1);` here, as you would otherwise get Watchdog resets in an I2C scan.

Reply:
The I2C scan now takes 14 seconds, which is way too long.
Not sure why, as it is way more than 9x the time it used to take.

Reply:
Ah found it.. the default settings set the low clock speed to 100, instead of 100 kHz.
I think that does make a difference :)

Reply:
I mostly tested on an ESP32, and didn't have issues there. Now I'm testing on ESP8266, and it is dead slow indeed. Think you've pinpointed the cause, below. Working on it.

Reply:
> 
> 
> Ah found it.. the default settings set the low clock speed to 100, instead of 100 kHz.
> I think that does make a difference :)

That is because I didn't update the Build number again after adding this new setting, I had the same issue on the ESP32 but changed it immediately to 100000 kHz, but on the ESP8266 I forgot to update/check the value, and the minimum input value is 100, that's why it was at 100 😬 In a normal update from below build 20110 it should be set at 100000 kHz.